src/Umbraco.Cms.Integrations.Crm.Dynamics/Api/Management/Controllers/CheckOAuthConfigurationController.cs

```diff
@@ -0,0 +1,40 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Options;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Umbraco.Cms.Integrations.Crm.Dynamics.Configuration;
+using Umbraco.Cms.Integrations.Crm.Dynamics.Models.Dtos;
+using Umbraco.Cms.Integrations.Crm.Dynamics.Services;
+
+namespace Umbraco.Cms.Integrations.Crm.Dynamics.Api.Management.Controllers
+{
+    public class CheckOAuthConfigurationController : FormsControllerBase
+    {
+        public CheckOAuthConfigurationController(IOptions<DynamicsSettings> options, DynamicsService dynamicsService, DynamicsConfigurationService dynamicsConfigurationService, DynamicsComposer.AuthorizationImplementationFactory authorizationImplementationFactory) : base(options, dynamicsService, dynamicsConfigurationService, authorizationImplementationFactory)
+        {
+        }
+
+        [HttpGet("check-oauth-configuration")]
+        [ProducesResponseType(typeof(OAuthConfigurationDto), StatusCodes.Status200OK)]
+        [ProducesResponseType(StatusCodes.Status404NotFound)]
```

Comment:
Doesn't look like this endpoint can return 404, so think this should be removed.

I won't repeat this comment in other cases, but would be worth just looking carefully over your endpoints and make sure they all have the correct settings for the `ProducesResponseType` attributes.